### PR TITLE
Format imports (#11)

### PR DESCRIPTION
**Pull request by therealppa**
I took a shot at #11 because I would also like to have that feature :)

### CODE REVIEW

**Comment by pastjean** [comment]
Thanks for that feature :) I'm gonna try to deploy it soon

File: src/test/resources/importclean/pom.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                            <goal>format</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
+
```
Comment (pastjean) [inline src/test/resources/importclean/pom.xml]: lot's a empty lines here

File: src/test/resources/importunsorted/pom.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
+
+
```
Comment (pastjean) [inline src/test/resources/importunsorted/pom.xml]: empty lines

File: src/test/resources/importunused/pom.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
+
+
```
Comment (pastjean) [inline src/test/resources/importunused/pom.xml]: strange empty lines again

**Comment by pastjean** [comment]
I deployed 1.7.0 :D have fun

**Comment by malaporte** [comment]
🎉  thanks @therealppa and @pastjean !

**Pull request merged**